snap/local/wrappers/fetch-oci

```diff
@@ -29,19 +29,15 @@ fi
 timeout 110s sh <<EOT || echo "Timed out waiting to install microk8s image."
 
 echo "Wait for microk8s to be ready if needed."
-wait_result=$(microk8s.status --wait-ready --timeout 30 2>&1)
-if [ $? -ne 0 ]; then
-    echo "${wait_result}"
-    exit 0
-fi
-
-juju_version=$(/snap/bin/juju version | rev | cut -d- -f3- | rev)
-oci_image="docker.io/jujusolutions/jujud-operator:$juju_version"
+microk8s.status --wait-ready --timeout 30 2>&1
```

Comment:
we don't need the result check?

Reply:
nah, the internal exit code won't be passed out.